rune/ValueRune.ts

```diff
@@ -53,6 +53,18 @@ export class ValueRune<out T, out U = never> extends Rune<T, U> {
     })
   }
 
+  handle<T2 extends T, T3, U2>(
+    guard: Guard<T, T2>,
+    alt: (rune: ValueRune<T2, never>) => Rune<T3, U2>,
+  ): ValueRune<Exclude<T, T2> | T3, U | U2> {
+    return ValueRune.new(
+      RunMatch,
+      this,
+      (x: T): x is T2 => checkGuard(x, guard),
```

Comment:
This should pass `guard` directly like in `RunUnhandle`

Reply:
This wasn't resolved

Reply:
oops my bad, I updated the Run function for this and forgot to update it here